packages/compass-smoke-tests/tsconfig.json

```diff
@@ -3,7 +3,11 @@
   "compilerOptions": {
     "noEmit": true,
     "types": ["node"],
-    "lib": ["ES2021"]
+    "lib": ["ES2021"],
+    "module": "NodeNext",
+    "paths": {
+      "compass-mongodb-com": ["./src/compass-mongodb-com.d.ts"]
```

Comment:
Needed for ts-node https://github.com/TypeStrong/ts-node?tab=readme-ov-file#missing-types